ee/tabby-ui/app/(dashboard)/settings/(integrations)/repository/[kind]/detail/components/update-github-provider-form.tsx

```diff
@@ -60,12 +62,19 @@ export const UpdateProviderForm: React.FC<UpdateProviderFormProps> = ({
   )
 
   const onSubmit = async (values: CreateRepositoryProviderFormValues) => {
-    await updateGithubRepositoryProvider({
+    const res = await updateGithubRepositoryProvider({
       input: {
         id,
         ...values
       }
     })
+    if (res?.data?.updateGithubRepositoryProvider) {
+      onUpdate?.()
+    } else {
+      toast.error(
```

Comment:
No need for manual toasting; if an error occurs, <FormMessage /> will render and display the error message